app/client/views/admin/users/adminUsersCtrl.js

```diff
@@ -130,6 +130,52 @@ angular.module('reg')
 
       };
 
+      $scope.acceptChecked = function(){
+        console.log($scope.users);
+        // Tracking info for mass admit
+        var numToAccept = 0;
+
+        // Stupid error handling
+        for (let i = 0; i < $scope.users.length; i++){
+          if($scope.users[i].markedForAcceptance) {
+            numToAccept += 1;
+          }
+        }
+        if (numToAccept == 0){
+          swal("Oops", "No one was checked", "info");
+          return;
+        }
+
+        // Stupid check pt. 2
+        swal({
+          title: "Whoa, wait a minute!",
+          text: "You are about to accept " + numToAccept + " users!",
+          type: "warning",
+          showCancelButton: true,
+          confirmButtonColor: "#DD6B55",
+          confirmButtonText: "Yeah, do it.",
+          closeOnConfirm: false
+        }, function() {
+          for (let i = 0; i < $scope.users.length; i++){
+            var user = $scope.users[i];
+            if(user.markedForAcceptance){
+              // I just log or err the callbacks but I should like count
+              // them up and show a sweet alert or something later
+              UserService
+                .admitUser(user._id)
+                .success(function(user){
+                  console.log(user.profile.name + " (" + user.email + ") Admitted!");
```

Comment:
I'm not sure if this is really something you need to worry about, but if the user can't be accepted, then I believe the function returns null, so this throws an error. For example, if the user hasn't filled out an application.

Reply:
Yeah fair, I was orginally trying to list everyone accepted and denied on the page for safety but ill just remove the success